### PR TITLE
[excel] (PivotTable) Adding PivotTable samples

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR gives samples to the PivotFilterType and PivotLayoutType enums.

### CODE REVIEW

File: docs/docs-ref-autogen/excel/excelscript/excelscript.pivotfield.yml [hunk 1]
```diff
@@ -89,7 +89,27 @@ methods:
...
+          ```TypeScript
+          /**
+           * This script gets the "Type" field from the "Farms Sales" PivotTable 
+           * and clears the value filter from it.
+           */ 
+          function main(workbook: ExcelScript.Workbook) {
+            // Get the PivotTable named "Farm Sales".
+            const pivot = workbook.getPivotTable("Farm Sales");
+
+            // Get the "Type" field
```
Comment (AlexJerabek) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.pivotfield.yml:106]: The change with this period didn't get pushed. Should be fixed in a minute.

File: docs/sample-scripts/excel-scripts.yaml [hunk 2]
```diff
@@ -2066,6 +2098,38 @@
         // Example log: "Grand total of Sum of Crates Sold Wholesale: 11000"
       });
     }
+'ExcelScript.PivotLayout#setLayoutType:member(1)':
+  - |-
+    /**
+     * This script sets the layout of "Farms Sales" PivotTable to the "tabular" setting.
+     * This places the fields from the Rows area in separate columns.
```
Comment (ElizabethSamuel-MSFT) [inline docs/sample-scripts/excel-scripts.yaml:2105]:
```suggestion
     * This script sets the layout of the "Farms Sales" PivotTable to the "tabular"
     * setting. This places the fields from the Rows area in separate columns.
```

File: docs/sample-scripts/excel-scripts.yaml [hunk 3]
```diff
@@ -2066,6 +2098,38 @@
...
+      const layout = pivot.getLayout();
+
+      // Set the layout type to "tabular".
+      layout.setLayoutType(ExcelScript.PivotLayoutType.tabular);
+    }
+'ExcelScript.PivotLayoutType:enum':
+  - |-
+    /**
+     * This script sets the layout of "Farms Sales" PivotTable to the "tabular" setting.
+     * This places the fields from the Rows area in separate columns.
```
Comment (ElizabethSamuel-MSFT) [inline docs/sample-scripts/excel-scripts.yaml:2121]:
```suggestion
     * This script sets the layout of the "Farms Sales" PivotTable to the "tabular"
     * setting. This places the fields from the Rows area in separate columns.
```

File: docs/sample-scripts/excel-scripts.yaml [hunk 4]
```diff
@@ -2197,6 +2261,22 @@
       pivotTable.addRowHierarchy(pivotTable.getHierarchy("Type"));
       pivotTable.addDataHierarchy(pivotTable.getHierarchy("Sales"));
     }
+'ExcelScript.PivotTable#getLayout:member(1)':
+  - |-
+    /**
+     * This script sets the layout of "Farms Sales" PivotTable to the "tabular" setting.
+     * This places the fields from the Rows area in separate columns.
```
Comment (ElizabethSamuel-MSFT) [inline docs/sample-scripts/excel-scripts.yaml:2268]:
```suggestion
     * This script sets the layout of the "Farms Sales" PivotTable to the "tabular"
     * setting. This places the fields from the Rows area in separate columns.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**